Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -35,6 +35,8 @@ + (UILocalNotification *)UILocalNotification:(id)json
   UILocalNotification *notification = [UILocalNotification new];
   notification.fireDate = [RCTConvert NSDate:details[@"fireDate"]] ?: [NSDate date];
   notification.alertBody = [RCTConvert NSString:details[@"alertBody"]];
+  notification.alertAction = [RCTConvert NSString:details[@"alertAction"]] ?: nil;
+  notification.applicationIconBadgeNumber = [RCTConvert NSInteger:details[@"applicationIconBadgeNumber"]] ?: 0;
```

Comment:
Reading the documentation added to the JS file, that does not appear to be the intention, so this should be more like:

```objc
if (details[@"applicationIconBadgeNumber"] != nil) {
  notification.applicationIconBadgeNumber = [RCTConvert NSInteger:details[@"applicationIconBadgeNumber"]];
}
```